Ecommerce/index.html

```diff
@@ -1,13 +1,78 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>E-commerce</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <!-- Start coding from the lines below for ECOMMERCE TASK -->
-    <!-- to import image from public folder, do it like this = <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
-</html>
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>About Us</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+        <style>
+            .top{
+                background-color:#4D133F;
+                width:100%;
+                height:11rem;
+                text-align:center;
+                color:#ffff;
+                font-size:0.8em;
+                margin-top:-1rem;
+            }
+            .box{
+              background-color:#E8E8E9;
+              width:100%;
+              height:100%;
+            }
+            .box-container{

```

Comment:
The border color for the input elements is not set correctly. It should be a valid color value, such as '#555' instead of '#5555'